src/lib/auth.js

```diff
@@ -78,3 +77,8 @@ function findUser(profile, accessToken, config, callback)  {
         return callback(false, username + ' is not authorized')
     }
 }
+
+function verifyApiKey(config, req)  {
+    var apiKey = req.headers['apikey'] || req.query['apikey'] || '';
```

Comment:
Authorization header: Sounds good. I will make the change
Regarding the query string param I don't feel strongly about this. What's your concern?